src/CasparCG.ts

```diff
@@ -77,460 +77,492 @@ import {
 	ByeParameters,
 	KillParameters,
 	RestartParameters,
+	InfoChannelParameters,
+	InfoLayerParameters,
 } from './parameters'
 
 export class CasparCG extends BasicCasparCGAPI {
-	async loadbg(params: LoadbgParameters): Promise<SendResult> {
+	async loadbg(params: LoadbgParameters): Promise<SendResult<CReturnType<Commands.Loadbg>>> {
```

Comment:
This return type could be simplified by creating a type for it. I.e. `export type APIRequest<C extends Commands> = Promise<SendResult<CReturnType<C>>>`